common/iveraisonclient.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import "github.com/veraison/apiclient/verification"
+
+type IVeraisonClient interface {
```

Comment:
Kindly add few lines of comment to state that IVeraisonClient is the Interface to Veraison api client, as provided in veraison/apiclient

Reply:
ok, done